action-scripts/brownie/scripts/report_gauge_add.py

```diff
@@ -0,0 +1,165 @@
+    If it fails, it tries to extract gauge address from the transaction input.
+
+    Then, it extracts gauge data from mainnet or jump to sidechains if needed.
+
+    :param transaction: transaction to parse
+    :return: dict with parsed data
+    """
+    network.disconnect()
+    network.connect("mainnet")
+    if transaction['to'] != FLATBOOK[ADDR_BOOK.search_unique("v4/GaugeAdder")]:
+        return
+
+    # Parse only gauge add transactions
+    if not any(method in transaction["contractInputsValues"] for method in GAUGE_ADD_METHODS):
+        return
+    # Find command and gauge address
+    command = transaction["contractMethod"]["name"]
+    gauge_address = None
+    for method in GAUGE_ADD_METHODS:
+        gauge_address = transaction["contractInputsValues"].get(method)
+        if gauge_address:
+            break
+    # If no gauge address found, try to extract it from the encoded data
+    if not gauge_address:
+        target_contract = Web3.toChecksumAddress(
+            transaction["contractInputsValues"]["target"]
+        )
+        try:
+            (command, inputs) = Contract(target_contract).decode_input(
+                transaction["contractInputsValues"]["data"])
+        except (VirtualMachineError, Exception):
+            print(f"Failed to decode input for transaction: {transaction}")
+            return
+        gauge_address = inputs[0]
+    if not gauge_address:
+        print("! Gauge address not found in transaction data")
+        return
+
+    # Finally, extract gauge data from mainnet or jump to sidechains if needed
+    gauge = Contract(gauge_address)
+    gauge_selectors = gauge.selectors.values()
+    gauge_cap = (
+        f"{gauge.getRelativeWeightCap() / 10 ** 16}%"
+        if "getRelativeWeightCap" in gauge_selectors else "N/A"
+    )
+    intersection_selector = list(set(gauge_selectors).intersection(list(SELECTOR_MAP.keys())))
+    if len(intersection_selector) > 0:  # Is sidechain
+        chain = SELECTOR_MAP[intersection_selector[0]]
+        recipient = gauge.getRecipient()
+        network.disconnect()
+        network.connect(f"{chain}-main")
+        sidechain_recipient = Contract(recipient)
+        if "reward_receiver" in sidechain_recipient.selectors.values():
+            sidechain_recipient = Contract(sidechain_recipient.reward_receiver())
+        pool_name, pool_symbol, pool_id, pool_address, a_factor, fee = get_pool_info(
+            sidechain_recipient.lp_token())
+        style = STYLE_CHILD_CHAIN_STREAMER
+    elif "name" not in gauge_selectors:
+        recipient = Contract(gauge.getRecipient())
+        escrow = Contract(recipient.getVotingEscrow())
+        pool_name, pool_symbol, pool_id, pool_address, a_factor, fee = get_pool_info(escrow.token())
+        style = STYLE_SINGLE_RECIPIENT
+    else:
+        (pool_name, pool_symbol, pool_id, pool_address, a_factor, fee) = get_pool_info(
+            gauge.lp_token())
+        style = STYLE_MAINNET
+
+    return {
```

Comment:
In gaugaddrv4 the chain name can be determined by the function call, but if it is not apparent, then it may not hurt to add that here as well, it does make the table wide.  Could ask the maxis and see what they think.